astropy/table/tests/test_jsviewer.py

```diff
@@ -234,23 +234,40 @@ def test_show_in_notebook():
         htmlstr_windx_named = t.show_in_notebook(show_row_index="realidx").data
         htmlstr_woindx = t.show_in_notebook(show_row_index=False).data
 
-        assert (
-            textwrap.dedent(
-                """
-        <thead><tr><th>idx</th><th>a</th><th>b</th></tr></thead>
-        <tr><td>0</td><td>1</td><td>b</td></tr>
-        <tr><td>1</td><td>2</td><td>c</td></tr>
-        <tr><td>2</td><td>3</td><td>a</td></tr>
-        <tr><td>3</td><td>4</td><td>d</td></tr>
-        <tr><td>4</td><td>5</td><td>e</td></tr>
-        """
-            ).strip()
-            in htmlstr_windx
-        )
-
-        assert (
-            "<thead><tr><th>realidx</th><th>a</th><th>b</th></tr></thead>"
-            in htmlstr_windx_named
-        )
-
-        assert "<thead><tr><th>a</th><th>b</th></tr></thead>" in htmlstr_woindx
+    assert (
+        textwrap.dedent(
+            """
+    <thead><tr><th>idx</th><th>a</th><th>b</th></tr></thead>
+    <tr><td>0</td><td>1</td><td>b</td></tr>
+    <tr><td>1</td><td>2</td><td>c</td></tr>
+    <tr><td>2</td><td>3</td><td>a</td></tr>
+    <tr><td>3</td><td>4</td><td>d</td></tr>
+    <tr><td>4</td><td>5</td><td>e</td></tr>
+    """
+        ).strip()
+        in htmlstr_windx
+    )
+
+    assert (
+        "<thead><tr><th>realidx</th><th>a</th><th>b</th></tr></thead>"
+        in htmlstr_windx_named
+    )
+
+    assert "<thead><tr><th>a</th><th>b</th></tr></thead>" in htmlstr_woindx
+
+
+@pytest.mark.skipif(not HAS_IPYDATAGRID, reason="requires ipydatagrid")
+# https://github.com/bqplot/bqplot/issues/1624 and such
+@pytest.mark.filterwarnings(r"ignore:((.|\n)*)traitlets((.|\n)*):DeprecationWarning")
+def test_show_in_notebook_ipydatagrid():
+    from ipydatagrid import DataGrid
+
+    t = Table()
+    htmlstr_windx = t.show_in_notebook(backend="ipydatagrid")
```

Comment:
Test that the default matches the documentation.
```suggestion
    htmlstr_windx = t.show_in_notebook()
```
Not really important, but the variable name `htmlstr_windx` is a bit odd and certainly implies that the return is a string. Maybe just call it `datagrid`.